<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_insert_date_season</class>
 <widget class="QDialog" name="Dialog_insert_date_season">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>ВТОРНИК:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_6"/>
     </item>
     <item row="3" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>ЧЕТВЕРГ:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="maxLength">
        <number>64</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>СРЕДА:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>ПЯТНИЦА:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Введите, пожалуйста, номер суточного профиля ПОНЕДЕЛЬНИК:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>СУББОТА:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>ВОСКРЕСЕНЬЕ:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEdit_7"/>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Выберите, пожалуйста, дату сезонного недельного профиля:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCalendarWidget" name="calendarWidget"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2" alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" alignment="Qt::AlignRight">
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>93</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Выбрать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
